Makefile

```diff
@@ -12,28 +13,37 @@ build: core $(all_apps)
 .PHONY: clean
 clean: clean-core $(addprefix clean-app-,$(apps))
 
+node_modules: package.json package-lock.json
+	npm install && touch node_modules
+
+core/css/uikit.%.css: src/themes/%.less node_modules
+	node_modules/less/bin/lessc src/themes/$*.less core/css/uikit.$*.css --relative-urls
+
+core/js/core.bundle.js: node_modules
```

Comment:
to be able to generate "core/js/core.bundle.js" we require  "node_modules" to exist, else we can't run npm.
this will trigger the "node_modules" rule above